packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -772,6 +774,7 @@ export class ModularChangeFamily
 		genId: IdAllocator,
 		crossFieldTable: InvertTable,
 		revisionMetadata: RevisionMetadataSource,
+		revision: RevisionTag | undefined,
```

Comment:
The call to `rebaser.invert` should be made to take this revision (assuming this represents the revision for the inverse change that is being generated).